utils/composer-packages/src/ValueObject/ExtensionVersion.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\ComposerPackages\ValueObject;
+
+use Rector\Composer\ValueObject\PackageAndVersion;
+use Stringable;
+use Webmozart\Assert\Assert;
```

Comment:
What is the difference from this to https://packagist.org/packages/beberlei/assert besides way more downloads on packagist

Reply:
I am using the webmozart/assert library since ages. Don´t know any real difference for the most common assertions.